tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -11,8 +11,9 @@
 #ifndef ROOT_RRESULTPTR
 #define ROOT_RRESULTPTR
 
+#include "ROOT/RActionBase.hxx"
+#include "ROOT/RLoopManager.hxx"
 #include "ROOT/TypeTraits.hxx"
-#include "ROOT/RDFNodes.hxx"
```

Comment:
Is ROOT compilable right before this commit? The renaming of a header and fixing the includes that use the old name should all be in a single commit, no?

Reply:
ROOT _was_ compilable at every commit of this PR before the changes to rdataframe's `CMakeLists.txt` in master.
Now ROOT is broken at every commit of this PR because the `CMakeLists` are out of sync.
I talked to Axel, I will do a `rebase` + `merge --no-ff` to indicate that things will only compile with the last commit -- without polluting the git log too much.

Reply:
I have been working on these changes for the last couple of weeks, so when I started rdataframe's cmakelists were still using globbing and I didn't have to update the list of headers and sources at every commit.

It's not a problem, it's just a pain now to make every commit of the PR compileable independently, hence the merge -no-ff to preserve the compileability of master

Reply:
Ah, sorry about that. Didn't know you were working on changes like this. I think it's fine to use a merge commit in this case, or if you want I can do the fixups needed.